src/main/java/core/basesyntax/impl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.dao.OperationDaoImpl;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.ReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ReaderServiceImpl implements ReaderService {
+    private static final String FILE_NAME
```

Comment:
do not hardcode the file paths into the services, what if you will need to read another file?
![image](https://user-images.githubusercontent.com/80981426/212967288-068deaa0-791a-4edc-ac68-21ab5401fe50.png)
In general, any field that is not a constant we should be getting from outside the class, in our case let's try to separate all of the functionality as much as we can and call all of the needed methods and pass returned values down the road in the main method